nodeup/pkg/model/docker.go

```diff
@@ -298,252 +298,24 @@ var dockerVersions = []packageVersion{
-		Architectures:  []architectures.Architecture{architectures.ArchitectureAmd64},
-		Version:        "18.09.9",
-		Source:         "https://download.docker.com/linux/centos/7/x86_64/stable/Packages/docker-ce-18.09.9-3.el7.x86_64.rpm",
-		Hash:           "f4be41bf8093c076462a9a2d7669d1b3158e4c3799759dbf9689b77de49385a8",
-		ExtraPackages: map[string]packageInfo{
-			"docker-ce-cli": {
-				Version: "18.09.9",
-				Source:  "https://download.docker.com/linux/centos/7/x86_64/stable/Packages/docker-ce-cli-18.09.9-3.el7.x86_64.rpm",
-				Hash:    "b1658ece6b8524a9c23a8623a7485b361c61a49ba887b51d9cc4ef58cfeb878a",
-			},
-		},
-	},
-
 	// 18.09.9 - Linux Generic
-	//
-	// * AmazonLinux2: the Centos7 package depends on container-selinux, but selinux isn't used on amazonlinux2
-	// * UbuntuFocal: no focal version available at download.docker.com
 	{
 		PackageVersion: "18.09.9",
 		PlainBinary:    true,
-		Distros:        []distros.Distribution{distros.DistributionAmazonLinux2, distros.DistributionFocal},
 		Architectures:  []architectures.Architecture{architectures.ArchitectureAmd64},
 		Source:         "https://download.docker.com/linux/static/stable/x86_64/docker-18.09.9.tgz",
 		Hash:           "82a362af7689038c51573e0fd0554da8703f0d06f4dfe95dd5bda5acf0ae45fb",
 	},
 
-	// 19.03.4 - k8s 1.17 - https://github.com/kubernetes/kubernetes/pull/84476
-
-	// 19.03.4 - Linux Generic
-	//
-	// * AmazonLinux2: the Centos7 package depends on container-selinux, but selinux isn't used on amazonlinux2
-	// * UbuntuFocal: no focal version available at download.docker.com
+	// 19.03.8 - Linux Generic
 	{
 		PackageVersion: "19.03.4",
 		PlainBinary:    true,
-		Distros:        []distros.Distribution{distros.DistributionAmazonLinux2, distros.DistributionFocal},
 		Architectures:  []architectures.Architecture{architectures.ArchitectureAmd64},
 		Source:         "https://download.docker.com/linux/static/stable/x86_64/docker-19.03.4.tgz",
 		Hash:           "efef2ad32d262674501e712351be0df9dd31d6034b175d0020c8f5d5c9c3fd10",
 	},
 
-	// 19.03.4 - Debian Stretch
-	{
-		PackageVersion: "19.03.4",
-		Name:           "docker-ce",
-		Distros:        []distros.Distribution{distros.DistributionDebian9},
-		Architectures:  []architectures.Architecture{architectures.ArchitectureAmd64},
-		Version:        "5:19.03.4~3-0~debian-stretch",
-		Source:         "https://download.docker.com/linux/debian/dists/stretch/pool/stable/amd64/docker-ce_19.03.4~3-0~debian-stretch_amd64.deb",
-		Hash:           "a5fedef212914c443ed71c9ba2fbe0cdf39e0a6e2da8dfcc29881c6c536877ce",
-		ExtraPackages: map[string]packageInfo{
-			"docker-ce-cli": {
-				Version: "5:19.03.4~3-0~debian-stretch",
-				Source:  "https://download.docker.com/linux/debian/dists/stretch/pool/stable/amd64/docker-ce-cli_19.03.4~3-0~debian-stretch_amd64.deb",
-				Hash:    "f0f3c9c91a9482b0fe120cd9e404c3ade342ce01d0d98a7f6bce3e16b7c57a11",
-			},
-		},
-	},
-
-	// 19.03.4 - Debian Buster
-	{
-		PackageVersion: "19.03.4",
```

Comment:
I don't think we should be removing these, since 19.03.4 is still a supported option in validation.go

Reply:
We don't remove it, it remains as static package for all distros.